lpr.Common/Interfaces/Services/IAuthService.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System.Security.Claims;
+using System.Threading.Tasks;
```

Comment:
Does this interface need this?

Reply:
Yea I get why you ask, 
Rider is a b*tich sometimes 
Because `Task<>` is used inside the interface as a return type. 
Rider throws an error if the `using` is not noted.